### PR TITLE
fix lack of parameters for admin.namespaces().splitNamespaceBundle

### PR DESCRIPTION
**Pull request by sunheyi6**
This PR Closes apache/pulsar#20801

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: docs/admin-api-namespaces.md [hunk 1]
```diff
@@ -523,7 +523,7 @@ pulsar-admin namespaces split-bundle --bundle 0x00000000_0xffffffff test-tenant/
 <TabItem value="Java">
 
 ```java
-admin.namespaces().splitNamespaceBundle(namespace, bundle)
+admin.namespaces().splitNamespaceBundle(namespace, bundle,unloadSplitBundles,splitAlgorithmName)
```
Comment (visortelle) [inline docs/admin-api-namespaces.md:526]: Maybe add spaces?

```suggestion
admin.namespaces().splitNamespaceBundle(namespace, bundle, unloadSplitBundles, splitAlgorithmName)
```

**Comment by shy-share** [comment]
@visortelle all add spaces done

**Comment by Anonymitaet** [comment]
@Demogorgon314 
Could you please help review this PR? Thank you!

File: versioned_docs/version-2.5.1/admin-api-namespaces.md [hunk 2]
```diff
@@ -612,7 +612,7 @@ $ pulsar-admin namespaces split-bundle --bundle 0x00000000_0xffffffff test-tenan
 
 ```java
 
-admin.namespaces().splitNamespaceBundle(namespace, bundle)
+admin.namespaces().splitNamespaceBundle(namespace, bundle, unloadSplitBundles, splitAlgorithmName)
```
Comment (Demogorgon314) [inline versioned_docs/version-2.5.1/admin-api-namespaces.md:615]: The `splitAlgorithmName ` parameter is added in version 2.5.1.

Comment (sunheyi6) [reply versioned_docs/version-2.5.1/admin-api-namespaces.md:615]: The modifications have been completed.

**Review by tisonkun — APPROVED** [review]
Thanks for your contribution!

**Review by Demogorgon314 — APPROVED** [review]

**Pull request merged**